src/components/AddButton.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styled from "styled-components";
+import Add from "../icons/Add";
+
+const ModalButton = styled.button`
```

Comment:
Why is this called `ModalButton`?  The `AddButton` do not depend on Modals. I propose to call it like `StyledButton`